packages/trader/src/Stores/Modules/Trading/Helpers/proposal.js

```diff
@@ -29,7 +30,7 @@ export const getProposalInfo = (store, response, obj_prev_contract_basis) => {
         ? { text: 'Payout', value: 'number_of_contracts' }
         : basis_list.find(o => o.value !== store.basis) || {};
 
-    const is_stake = contract_basis.text === 'Stake';
+    const is_stake = contract_basis.text === localize('Stake');
```

Comment:
can we use `contract_basis.value` rather than using localize here. The value is constant for any language

Reply:
unfortunately, your suggestion doesn't work for me because contract_basis.value is not localized.

Reply:
The valtes are fixed to `stake` & `payout`. Only text is localized

Reply:
@likhith-deriv this is the reason why the issue i fixed was happening: is_stake should give true for values to appear instead of 0.00 usd. So since contract_basis.text is always translated, and the issue is that we cannot compare it to contract_basis.value because it's not localized (translated)🙂

Reply:
contract_basis.value === 'stake' is working too ) and i like this better  🙂 thank you!